README.md

```diff
@@ -15,5 +15,5 @@ are building your own bot. But if you do, check out [CONTRIBUTING.md](CONTRIBUTI
 
 ## License
 
-Copyright (c) 2011-2014 GitHub, Inc. See the LICENSE file for license rights and
+Copyright (c) 2011-2015 GitHub, Inc. See the [LICENSE](https://github.com/github/hubot/blob/master/LICENSE) file for license rights and
```

Comment:
You can actually use relative links here. The benefit is that it doesn't hard-code the branch name, so it links to the right version if you are browsing a different branch.